src/config.jsonc

```diff
@@ -0,0 +1,135 @@
+// Configuration for MDTF-diagnostics driver script self-test.
```

Comment:
@wrongkindofdoctor - this complete file should probably not be included in the PR, correct?  Only the POD-relevant changes I believe.

Reply:
@jkrasting @jcstarr Correct. Any customized config files that developers want to retain should be in the POD directory.

Reply:
Is there something I should do to exclude this file or will it be handled in the review/approval process?

Reply:
@jcstarr You can remove the file from the remote branch only:

```
git rm --cached src/config.jsonc
git commit -m "commit message"
git push
```

If you want to retain it in the PR for reference, you can then move the file to diagnostics/TC_MSE in your local repo, commit it, and push the change to your branch.

Reply:
Oh okay thank you! I will just remove it from the remote branch using those steps.